pom.xml

```diff
@@ -1156,6 +1157,9 @@
                             <goals>
                                 <goal>sign</goal>
                             </goals>
+                            <configuration>
+                                <signer>bc</signer>
+                            </configuration>
```

Comment:
This way the plugin uses a Bouncy Castle's Java implementation of a GPG signer, rather than relying on a separate binary. Also simplifies local signing.